### PR TITLE
Lint fix issues

### PR DESCRIPTION
**Pull request by pw-ppodhajski**
Fix issues before add Github Actions with linters

### CODE REVIEW

File: mypy.ini [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[mypy]
+python_version = 3.10
+exclude = examples
+ignore_missing_imports = True
+follow_imports_for_stubs = True
+
+[mypy-pathway.*]
+check_untyped_defs = True
```
Comment (KamilPiechowiak) [inline mypy.ini:8]: I am also not sure if we want to check pathway with a stricter setting than on the default level.

File: mypy.ini [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[mypy-pandas.*] 
+follow_imports = skip
+
+[mypy-experimental.*]
+ignore_errors = True
```
[the remark this answers is not in the document]
Comment (pw-ppodhajski) [reply mypy.ini]: done

File: mypy.ini (hunk 1 above)
Comment (pw-ppodhajski) [reply mypy.ini]: done

**Review by KamilPiechowiak — APPROVED** [review]

File: mypy.ini (hunk 1 above)
Comment (embe-pw) [reply mypy.ini]: We want to have `check_untyped_defs = True` globally.

File: mypy.ini
Comment (embe-pw) [inline mypy.ini:1]: Let's have this inside `pyproject.toml`

Comment (pw-ppodhajski) [reply mypy.ini:1]: I'll add it here: https://github.com/pathwaycom/llm-app/pull/48

**Pull request merged**